lib/python/qmk/cli/__init__.py

```diff
@@ -8,6 +8,7 @@
 from . import doctor
 from . import hello
 from . import json
+from . import list_keyboards
```

Comment:
This should be structured slightly differently, as `qmk.cli.list.keyboards` instead.